frontend/api/login.ts

```diff
@@ -0,0 +1,23 @@
+import { LOGIN_URL } from './api';
+import axios, { AxiosError } from 'axios';
+
+interface LoginRequest {
+  email: string;
+  password: string;
+}
+
+interface LoginResponse {
+  email: string;
+  name: string;
+  authToken: string;
+}
+
+export const login = async (params: LoginRequest) => {
+  try {
+    const res = await axios.post<LoginResponse>(LOGIN_URL, params);
+    return res.data;
+  } catch (error) {
+    const err = error as AxiosError<{ message: string }>;
+    throw new Error(err.response?.data?.data?.message || '로그인 중 오류가 발생했습니다.');
+  }
```

Comment:
[타입] 여기도 조건문을 통해서 개선해볼 수 있겠네요. axios가 제공하는 isAxiosError를 사용해볼 수 있어요

```suggestion
    if (axios.isAxiosError<{ data: { message: string } }>(error)) {
      console.error('주문 요청 실패:', error.response?.data);
      throw new Error(error.response?.data?.data?.message);
    } else {
      console.error('주문 요청 실패:', error);
      throw new Error('주문에 실패했습니다.');
    }
  }
```

Reply:
네 수정하였습니다.